Makefile

```diff
@@ -1,2 +1,56 @@
+all: whatdoesitdo tests authors
+
 whatdoesitdo:
+	@# generate the README "What does it do?" section
 	@./gen-readme.sh
+
+# To decrease tests verbosity, comment out unneeded targets
+tests: downloadffprefs checknotcovered checkdeprecated stats cleanup
```

Comment:
This will fail horribly if you have e.g. ```MAKEFLAGS="-j 4"```, like [I do](https://github.com/pyllyukko/dotfiles/blob/master/bashrc#L55).

I'm not entirely sure what is the proper way to fix it, as I'm not that familiar with makefiles. Maybe with [.NOTPARALLEL](https://www.gnu.org/software/make/manual/html_node/Special-Targets.html#Special-Targets)?